src/crimson/common/interruptible_future.h

```diff
@@ -1096,6 +1096,9 @@ struct interruptor
 public:
   using condition = InterruptCond;
 
+  template <typename T>
+  using future = interruptible_future<InterruptCond, T>;
+
```

Comment:
can we rename it as ifuture or something else, future always is thought as seastar::future.

Reply:
ignore it, since it is only used in this structure, should be fine.